app/controllers/use_case_controller.rb

```diff
@@ -54,6 +57,19 @@ def update
 
   protected
 
+  def attribute_changes?
+    !@model_instance.changed_attributes.blank? || self.approver_email_changed?
+  end
+
+  def approver_email_changed?
+    email = params[:approver_email]
+    if !email.nil?
+      @model_instance.approval_changed?(email)
+    else
+      false
+    end
+  end
```

Comment:
I think this logic is specific to NCR, right? How about, in `UseCaseController`:
```ruby
def attribute_changes?
    !@model_instance.changed_attributes.blank?
end
```

Then, in the `Ncr::WorkOrderController`
```ruby
def attribute_changes?
    super || @model_instance.approval_changed?(@approver_email)
end
```